File: android studio/app/src/main/java/android/fit/ba/rentacar/data/RentalsResultVM.java
package android.fit.ba.rentacar.data;

import java.io.Serializable;
import java.util.Date;
import java.util.List;

public class RentalsResultVM implements Serializable {
    public static class Row implements Serializable {
        public int UserId;
        public String CarName;
        public Date ToDate;
        public int RentalId;
        public int CarId;
        public String Price;
        public Date FromDate;
    }

    public List<Row> rows;
}

File: web/eRentACar/ModelVM/RentalsResultVM.cs
using System;
using System.Collections.Generic;

namespace eRentACar.ModelVM
{
    public class RentalsResultVM
    {
        public List<Row> rows { get; set; }

        public class Row
        {
            public int UserId { get; internal set; }
            public string CarName { get; internal set; }
            public DateTime ToDate { get; internal set; }
            public int RentalId { get; internal set; }
            public int CarId { get; internal set; }
            public string Price { get; internal set; }
            public DateTime FromDate { get; internal set; }
        }
    }
}